Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -80,14 +80,17 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
       |> \.dataSource .~ self.dataSource
       |> \.delegate .~ self
       |> \.tableHeaderView .~ self.projectNavigationSelectorView
-      |> \.tableFooterView .~ nil
+      |> \.tableFooterView .~ UIView(frame: .zero)
     self.tableView.registerCellClass(ProjectFAQsAskAQuestionCell.self)
     self.tableView.registerCellClass(ProjectFAQsCell.self)
     self.tableView.registerCellClass(ProjectFAQsEmptyStateCell.self)
     self.tableView.registerCellClass(ProjectFAQsHeaderCell.self)
     self.tableView.registerCellClass(ProjectEnvironmentalCommitmentCell.self)
     self.tableView.registerCellClass(ProjectEnvironmentalCommitmentDisclaimerCell.self)
     self.tableView.registerCellClass(ProjectEnvironmentalCommitmentHeaderCell.self)
+    self.tableView.registerCellClass(ProjectRisksCell.self)
```

Comment:
Unsure of how to test these cells. The ProjectPageViewControllerTests snapshots aren't rendering any cells because we default to overview and that is blank right now. I tried calling projectNavigationDidSelect but that requires a reference to the `ProjectNavigationSelectorView` which is privately declared in the view controller file.

Reply:
Cool, sounds good. ty!